src/main/java/task/second/ExecutorServiceSolutionTest.java

```diff
@@ -0,0 +1,49 @@
+package task.second;
+
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ExecutorServiceSolutionTest {
```

Comment:
this class should it package with test not in main

Reply:
done